kolena/initialize.py

```diff
@@ -96,3 +136,20 @@ def initialize(
         # Configure third party logging based on verbosity
         pd.set_option("display.max_colwidth", None)
         log.info(f"connected to {get_platform_url()}")
+
+
+def _find_token() -> str:
+    if KOLENA_TOKEN_ENV in os.environ:
+        return os.environ[KOLENA_TOKEN_ENV]
+
+    hostname = urlparse(state._get_api_base_url()).hostname
+    try:
+        netrc_file = netrc.netrc()
+        record = netrc_file.authenticators(hostname)
+        if record and record[2]:
+            return record[2]
+        raise MissingTokenError
+    except Exception:
+        raise MissingTokenError(
+            f"No api token in `{KOLENA_TOKEN_ENV}` env variable or in .netrc file under {hostname}",
+        )
```

Comment:
Is this blanket exception catch and re-raise necessary? It leads to a more convoluted stacktrace than necessary ("During handling of the above exception, another exception occurred...").

Reply:
No, definitely not. I debated this portion of the code for a bit before settling on this. I'll rethink the handling here